docs/sources/operations/troubleshooting.md

```diff
@@ -99,8 +99,10 @@ port (`9080` or `3101` if using Helm) locally:
 
 ```bash
 $ kubectl port-forward loki-promtail-jrfg7 9080
-# Then, in a web browser, visit http://localhost:9080/service-discovery
-```
+``````
```

Comment:
I think there should only be three back-ticks.

Reply:
Good catch, thanks!